assets/css/global.scss

```diff
@@ -56,3 +56,16 @@ $promo_highlight: #43af99;
 	color: #2c3338;
 	font-weight: 400;
 }
+
+.sensei-upsell-pro-badge {
```

Comment:
For consistency, it may be better to leverage the CSS that's already being used for the Pro badge next to Groups in the sidebar:

![Screenshot 2023-03-20 at 9 55 20 AM](https://user-images.githubusercontent.com/1190420/226361438-644b02d5-1f99-4fc6-a5d0-6aed13826bbf.jpg)

Reply:
I tried to reuse some of the common styles between them [here](https://github.com/Automattic/sensei/pull/6668/commits/21b67e002a0dbb4b053f911a715a6a23278f1849), but most of the styles of the "Pro" badge on "Groups" in the menu comes from core WP https://github.com/WordPress/WordPress/blob/136dc2b108395d98f613df9f7fd26e3d9593b0cd/wp-admin/css/admin-menu.css#L432, and are bound with #adminmenu selector. So probably can't fully reuse that here